config/suppressions.xml

```diff
@@ -79,9 +79,11 @@
     <suppress checks="ClassDataAbstractionCoupling" files="(CheckerTest|TreeWalkerTest|BaseCheckTestSupport|AbstractModuleTestSupport|XdocsPagesTest|CheckstyleAntTaskTest)\.java"/>
     <suppress checks="ClassDataAbstractionCoupling" files="PropertyCacheFile\.java"/>
     <suppress checks="ClassDataAbstractionCoupling" files="PropertyCacheFileTest\.java"/>
+    <!-- Class Fan-Out Complexity is more 25. Setting in https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L374 -->
```

Comment:
This comment explains nothing, it should say why can't we just make complexity 25 or lower. As we discussed privately: it is OK to reuse an existing comment if it fits, so you can just add new suppression to the same group without an additional comment.

Reply:
fixed